idflakies-maven-plugin/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>edu.illinois.cs</groupId>
+        <artifactId>idflakies</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>idflakies-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
+
+    <name>iDFlakies Maven Plugin</name>
+    <description>Maven plugin for iDFlakies</description>
+
+    <developers>
+        <developer>
+            <name>Wing Lam</name>
+            <email>winglam2@illinois.edu</email>
+        </developer>
+        <developer>
+            <name>Reed Oei</name>
+            <email>reedoei2@illinois.edu</email>
+        </developer>
+        <developer>
+            <name>August Shi</name>
+            <email>awshi2@illinois.edu</email>
+        </developer>
+    </developers>
+
+    <licenses>
+        <license>
+            <name>University of Illinois/NCSA Open Source License</name>
+            <url>https://opensource.org/licenses/NCSA</url>
+        </license>
+    </licenses>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+              <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
This is defined under pluginManagement, so I'm not sure if it is actually be used properly.

Reply:
To test, remove entire plugin and inspect compilation?